In [8]:
#import packages
import numpy as np
import scipy.stats as stats

In [13]:
#input data
t_amount = np.array([100,150,50,100,130,120,100,110,75,65,150,120,50,100,100,140,90,150,50,90])
t_amount

array([100, 150,  50, 100, 130, 120, 100, 110,  75,  65, 150, 120,  50,
       100, 100, 140,  90, 150,  50,  90])

**1. Hypotheses (H0 and H1)**

H0 : The average of transactions’ amount after the campaign = $100

H1 : The average of transactions’ amount after the campaign > $100

**2. Significance level** :

alpha = 0.05

**3. Calculate T-statistic and P-value**

In [14]:
#perform one sample t-test
stats.ttest_1samp(a=t_amount,popmean=100)

Ttest_1sampResult(statistic=0.2731657711666385, pvalue=0.787672894169515)

In [10]:
0.787672894169515/2

0.3938364470847575

Using One tailed test

From the calculation, we get :

t statistic = 0.2732 and p-value obtained = 0.3938

**4. Critical Value (T-Table) and Critical Region**

In [11]:
stats.t.ppf(1-0.05,19)

1.729132811521367

 Critical Region: t > 1.729
 
 Decision Rule
 - If using alpha
     - if P-value < alpha (0.05), then we reject H0
     - if P-value > alpha (0.05), then we fail to reject H0
 - If using t statistic
     - if t statistic > t critical (1.729), then we reject H0
     - if t statistic < t critical (1.729), then we fail to reject H0

**Comparing and Make Conclusion**

Decision

- Using alpha : Because P-value/2 (0.3938) > alpha (0.05), then we fail to reject H0
- Using t statistic : Because t statistic (0.2732) < t critical (1.729), then we fail to reject H0

Conclusion

- With a significance level of 5%, it can be concluded that The average of transactions' amount after the campaign = $100. The new campaign that is run does not increase the average of transactions' amount, the new campaign is not effective.